CMakeLists.txt

```diff
@@ -524,6 +524,7 @@ option(ENABLE_COVERAGE "Coverage is enabled" OFF)
 option(PG_DEBUG_REFS "PG Ref debugging is enabled" OFF)
 
 option(WITH_TESTS "enable the build of ceph-test package scripts/binaries" ON)
+option(CTEST_TIMEOUT "Allow customizing the test timeout" 3600)
```

Comment:
@wjwithagen unlike the ON/OFF options, this option is different in that it allows a string parameter and its description should reflect this difference. so i'd suggest put something like

```cmake
set(CEPH_TEST_TIMEOUT 1 CACHE STRING
  "Maximum time allowed before CTest kills a test")
```

also please avoid using names with prefix of `CMAKE` or `CTEST`. to avoid unnecessary confusion. for instance, https://cmake.org/cmake/help/v3.6/variable/CTEST_TEST_TIMEOUT.html .

Reply:
@tchaikov 
Ping?